Makefile

```diff
@@ -7,6 +7,7 @@ build:
 	-X 'main.mongousr=${MONGO_USR}' \
 	-X 'main.mongopw=${MONGO_PW}' \
 	-X 'main.spaceapi=${SERVICES_API_URL}' \
+	-X 'main.spacestoragesiteurl=${SPACE_STORAGE_SITE_URL}' \
```

Comment:
I think we also need this added to the github actions files (.github/workflows/release.yml and .goreleaser.yml) (and also PRD vs DEV if they will be diff).

Reply:
@jsonsivar Thanks for pointing that out. I've added it to the release and goreleaser yaml files.